package.json

```diff
@@ -93,6 +93,7 @@
     "redux-devtools-dock-monitor": "^1.0.1",
     "redux-devtools-log-monitor": "^1.0.1",
     "redux-thunk": "^1.0.3",
+    "sinon": "^1.17.3",
```

Comment:
Not sure that sinon is necessary. By removing process.exit and using webpack emitter to send messages from the loaders, we can avoid console and so skip sinon usage.